webapp-django/crashstats/crashstats/views.py

```diff
@@ -1118,6 +1119,22 @@ def crashes_per_day(request, default_context=None):
     context['graph_data'] = graph_data
     context['report'] = 'daily'
 
+    errors = []
+    for error in results.get('errors', []):
+        if not error['type'] == 'shards':
+            continue
+
+        week = int(error['index'][-2:])
+        year = int(error['index'][-6:-2])
+        day = isoweek.Week(year, week).monday()
+        percent = error['shards_count'] * 100 / 5
```

Comment:
5 is the number of shards per index in our Elasticsearch database. It is kind of a magical number. I'll move to configuration, it will make more sense like that.